src/datasources/balances-api/balances-api.manager.spec.ts

```diff
@@ -158,18 +148,26 @@ describe('Balances API Manager Tests', () => {
   });
 
   describe('getFiatCodes checks', () => {
-    it('should return the intersection of all providers supported currencies', () => {
-      zerionBalancesApiMock.getFiatCodes.mockReturnValue(['EUR', 'GBP', 'ETH']);
+    it('should return the intersection of all providers supported currencies', async () => {
+      zerionBalancesApiMock.getFiatCodes.mockResolvedValue([
```

Comment:
`getFiatCodes` is now an `async` function, as getting the fiat codes for Coingecko implies a (cached) HTTP call.